.github/workflows/tests.yml

```diff
@@ -24,7 +24,10 @@ jobs:
   end-to-end-kustomize:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
```

Comment:
why did we not need go before?

Reply:
This is in my attempt to keep the environment consistent for `make helm` in CI and locally until we introduce a docker image to build helm template :-))))